RealmBrowser/Classes/RLMApplicationDelegate.m

```diff
@@ -94,6 +96,27 @@ - (BOOL)application:(NSApplication *)application openFile:(NSString *)filename
     return YES;
 }
 
+- (void)application:(NSApplication *)sender openFiles:(NSArray *)filenames
+{
+    if (filenames.count > kMaxNumberOfFilesAtOnce) {
+        NSString *message = [NSString stringWithFormat:@"Are you sure you wish to open all %lu Realm files?", (unsigned long)filenames.count];
+        
+        NSAlert *alert = [[NSAlert alloc] init];
+        [alert setMessageText:message];
+        [alert setInformativeText:@"Opening too many files at once may result in system instability."];
```

Comment:
Not sure "System instability" is the right wording?
Does your whole computer really get instable?
Or is it only the app?
Why 20 as a max? It could be 50? 100? depending on the systems open files?
Is it more that we think it's not on purpose that people try to open all those files at once, and we just want to help them avoid doing that by accident?